client/Register.jsx

```diff
@@ -0,0 +1,97 @@
+import React, {PropTypes, Component} from 'react';
+
+export default class Register extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentUsername: '',
+      currentPassword: '',
+      currentConfirmation: '',
+      message: ''
+    };
+
+    this.rejectUserTakenUsername = this.rejectUserTakenUsername.bind(this);
+    this.rejectUserBadPassword = this.rejectUserBadPassword.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.addUser = this.addUser.bind(this);
+    this.handleUsernameInput = this.handleUsernameInput.bind(this);
+    this.handlePasswordInput = this.handlePasswordInput.bind(this);
+    this.handleConfirmPasswordInput = this.handleConfirmPasswordInput.bind(this);
+    this.handleSuccess = this.handleSuccess.bind(this);
+    this.passwordTest = this.passwordTest.bind(this);
+  }
+
+  addUser() {
+    $.ajax({
+      url: 'http://localhost:3000/users',
```

Comment:
this should be a relative path. hardcoding to localhost:3000 will break deployment.

please change to just /users